### PR TITLE
[TD-311] New helm installation proccess updates

### PR DESCRIPTION
**Pull request by vesko-tyk**

### CODE REVIEW

File: tyk-pro/README.md [hunk 1]
```diff
@@ -2,32 +2,43 @@
...
-
-	helm install tyk-pro ./tyk-pro -n tyk --wait
-
+```
+helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
+helm repo update
+```
+Before we proceed with installation of the chart we need to set some custom values. To see what options are configurable on a chart and save that options to a custom values.yaml file run:
+```
+helm show values tyk-helm/tyk-pro > values.yaml
```
Comment (letzya) [inline tyk-pro/README.md:11]: @vesko-tyk Can we update the instructions of the READMEs 
```
helm show readme tyk-helm/tyk-pro
helm show readme tyk-helm/tyk-hybrid
helm show readme tyk-helm/tyk-headless
```

File: README.md [hunk 2]
```diff
@@ -28,6 +20,16 @@ Another option for Redis and MongoDB could be charts provided by Bitnami:
 
 *Important Note regarding MongoDB:* This helm chart enables the PodDisruptionBudget for MongoDB with an arbiter replica-count of 1.  If you intend to perform system maintenance on the node where the MongoDB pod is running and this maintenance requires for the node to be drained, this action will be prevented due the replica count being 1.  Increase the replica count in the helm chart deployment to a minimum of 2 to remedy this issue.
 
+Another option for Redis and MongoDB, if you want to get started quickly is to use mongo.yaml and redis.yaml manifests located inside deploy/dependencies folder in this repository.
+**Please note that these provided manifests must not ever be used in production and for anything but a quick start evaluation only, use external DBs or Official Helm charts for MongoDB and Redis in any other case.**
+We provide these manifests so you can quickly have Tyk running however they are not meant for long term storage of data for example.
+
+	kubectl create namespace tyk
+	kubectl apply -f deploy/dependencies/mongo.yaml -n tyk
+	kubectl apply -f deploy/dependencies/redis.yaml -n tyk
```
Comment (abhishiekgowtham) [inline README.md:29]: I'm thinking it's better to remove these deps entirely from the repo, and link to using our simple mongo/redis helm charts.

File: tyk-headless/README.md [hunk 3]
```diff
@@ -6,4 +6,6 @@ Provides a Tyk Installation for Community Edition Gateways
 [![Install Tyk on Kubernetes](https://img.youtube.com/vi/mkyl38sBAF0/0.jpg)](https://www.youtube.com/watch?v=mkyl38sBAF0)
 
 ## Installation
-	helm install tyk-ce ./tyk-headless -n tyk
+	helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
+	helm repo update
+	helm install tyk-ce tyk-helm/tyk-headless -n tyk
```
Comment (abhishiekgowtham) [inline tyk-headless/README.md:11]: I find it useful to add `--version` option. Also, it'd be helpful once we start releasing often.

**Review by abhishiekgowtham — CHANGES_REQUESTED** [review]
Overall, looks good to me. Please review some of the feedback.

File: tyk-pro/README.md (hunk 1 above)
Comment (vesko-tyk) [reply tyk-pro/README.md:11]: This PR is still not merged, once approved and merged I'll update the actual package instructions as well.

File: README.md (hunk 2 above)
Comment (vesko-tyk) [reply README.md]: Agree, dependencies are now removed.

File: tyk-headless/README.md (hunk 3 above)
Comment (vesko-tyk) [reply tyk-headless/README.md]: That's done now.

**Review by abhishiekgowtham — APPROVED** [review]
/lgtm

File: tyk-pro/README.md [hunk 4]
```diff
@@ -2,32 +2,43 @@
...
+```
+helm install tyk-pro tyk-helm/tyk-pro --version 0.9.0 -f values.yaml -n tyk --wait
+```
 > Please note the --wait argument is important for successful dashboard bootstrap.
 
 Follow the instructions in the Notes that follow the installation to find your Tyk login credentials.
 
 ### Tyk Developer Portal
-You can enable the bootstraping of the Developer Portal using the `portal.bootstrap` option in `values.yaml`.
+You can enable the bootstrapping of the Developer Portal using the `portal.bootstrap` option in `values.yaml`.
```
Comment (letzya) [inline tyk-pro/README.md:25]: @vesko-tyk change it to "disable" since it's on by default

Comment (vesko-tyk) [reply tyk-pro/README.md:25]: Thanks @letzya. This is now updated on master.

**Pull request merged**